dependency_links.txt

```diff
@@ -1,2 +1 @@
 https://github.com/SciLifeLab/statusdb/archive/master.zip#egg=statusdb
```

Comment:
@guillermo-carrasco : Hej yes, it is still needed as I dint find the time to put `statusDB` in `Pypi` yet :sweat_smile: